src/assets/stylesheets/index.scss

```diff
@@ -3114,6 +3114,31 @@ img.cookie-img {
   }
 }
 
+/* list *
```

Comment:
* [ ] FMI Was this the original comment? Looks a bit sparse. 😛

Reply:
Both comments are by our css designer.